### PR TITLE
MSP-2764: add cel validation for jobtelemetry and shared memory

### PR DESCRIPTION
**Pull request by Uburro**

### CODE REVIEW

File: api/v1/slurmcluster_types.go [hunk 1]
```diff
@@ -333,6 +333,7 @@ type SlurmNodeControllerVolumes struct {
 // If CgroupVersion is "v2", InitContainer must not be null
```
Comment (asteny) [inline api/v1/slurmcluster_types.go]: I think that comments should be remover and all comments about InitContainer

Comment (Uburro) [reply api/v1/slurmcluster_types.go]: it has already removed in previous PR

Comment (asteny) [reply api/v1/slurmcluster_types.go]: so, you need to rebase?

Comment (Uburro) [reply api/v1/slurmcluster_types.go]: done =)

**Pull request merged**